packages/frontend/src/lib/upstream/Util.ts

```diff
@@ -0,0 +1,30 @@
+/**********************************************************************
+ * Copyright (C) 2023-2024 Red Hat, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ ***********************************************************************/
+
+export function capitalize(text: string): string {
+  return text.charAt(0).toUpperCase() + text.slice(1);
+}
+
+export function getTabUrl(routerPath: string, tabUrl: string): string {
+  const baseURL = routerPath.substring(0, routerPath.lastIndexOf('/'));
+  return `${baseURL}/${tabUrl}`;
+}
+
+export function isTabSelected(routerPath: string, tabUrl: string): boolean {
+  return routerPath === getTabUrl(routerPath, tabUrl);
+}
```

Comment:
Some unit tests maybe ?

Reply:
I copied this from PD, where there were no tests 😬. I've added some, and will PR to PD as well.